backend/api/management/commands/generate_fixtures.py

```diff
@@ -191,11 +191,11 @@ def create_basic_cp(self, cp_status):
             status=task_status,
         )
         if task_status == ComputeTask.Status.STATUS_DONE:
-            factory.create_performance(
-                test_task,
-                self.metric_function,
-                identifier="performance",
-            )
+            for task_output in test_task.outputs:
```

Comment:
Won't it create performance for `local`, `model` or `shared` ?

Reply:
test tasks normally only have performances as output, at least in this factory :)